docs/src/main/sphinx/admin/dynamic-filtering.rst

```diff
@@ -157,6 +155,36 @@ processed after a dynamic filter is pushed down to the table scan.
     "inputPositions" : 28800991,
     "dynamicFilterSplitsProcessed" : 1,
 
+Dynamic filters collected by the coordinator are also reported as a part of the
```

Comment:
drop "collected by the coordinator" and "also"

Reply:
done